net-libs/libmodbus/libmodbus-3.0.8.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="A groovy modbus library"
+DESCRIPTION="
+libmodbus is a free software library to send or receive data with a device which \
+respects the Modbus protocol. This library can use a serial port or an Ethernet  \
+connection.
+
+The functions included in the library have been derived from the Modicon Modbus  \
+Protocol Reference Guide which can be obtained from www.modbus.org.
+"
+HOMEPAGE="https://github.com/stephane/libmodbus"
+COPYRIGHT="
+	Stéphane Raimbault
+	Tobias Doerffel
+	Florian Forster
+	oldfaber
+	Hannu Vuolasaho
+	Michael Heimpold
+	Jimmy Bergström
+	Jakob Bysewski
+	"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/stephane/libmodbus/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="0e236c3d4fe0a7a3f7c6292a3d1183c584b03bb06154e0b95e71bc763559ac5d"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="5.0.5"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+
+PROVIDES="
+	libmodbus$secondaryArchSuffix = $portVersion
+	lib:libmodbus$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libmodbus${secondaryArchSuffix}_devel = $portVersion
+	devel:libmodbus$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	libmodbus$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	autogen.sh
+	LIBS=-lnetwork runConfigure ./configure
```

Comment:
Didn't check this, but maybe adding "network" to the next line in configure.ac could remove the need to add it here?

https://github.com/stephane/libmodbus/blob/master/configure.ac#L105

Reply:
@Begasus right now i'm avoid making patch because i've been tell to libmodbus dev about this issue, i hope they make some change so we will not need make any patch.